.gitignore

```diff
@@ -277,3 +277,6 @@ TSWLatexianTemp*
 
 # DS_Store file from Mac computers
 .DS_Store
+
+#IDEA project settings and configuration files
+.idea/
```

Comment:
What is this doing?

Reply:
For anyone using Intellij it makes sure IDE specific project setting changes are not auto committed

Reply:
Wait are you using Intellij?

Reply:
But you can merge this for now since it doesn't affect anything